content/articles/routes-and-navigators-in-flutter-apps/index.md

```diff
@@ -0,0 +1,274 @@
+ const SizedBox(
+              height: 30,
+            ),
+ ElevatedButton(
+              onPressed: () {
+ Navigator.pushNamed(context,Thirdscreen.routeName);
+              },
+              child: const Text('Go to third screen'),
+            ),
+ const SizedBox(
+              height: 10,
+            ),
+ ElevatedButton(
+              onPressed: () {
+ Navigator.pop(context);
+              },
+              child: const Text('Go to back'),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+```
+### Step5: Coding Third Screen
+Let's create the third screen. dart. This is the last page of our flutter mobile application. It also contains text and two buttons to navigate to the first and second screens. The first button uses the `navigator. push(context)` to help navigate to the first screen while the second button uses the `navigator. pop(context)` to navigate back to the previous page.   
+```dart
+import 'allfiles.dart';
+
+class Thirdscreen extends StatelessWidget {
+ static const routeName = '/Thirdscreen';
+ const Thirdscreen({Key? key}) : super(key: key);
+
+ @override
+ Widget build(BuildContext context) {
+ return Scaffold(
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+ const Text('This is the Third Screen'),
+ const SizedBox(
+              height: 30,
+            ),
+ ElevatedButton(
+              onPressed: () {
+ Navigator.of(context).push(MaterialPageRoute(
+                    builder: (context) => const FirstScreen()));
+              },
+              child: const Text('Go to firstScreen'),
+            ),
+ const SizedBox(
+              height: 10,
+            ),
+ ElevatedButton(
+              onPressed: () {
+ Navigator.pop(context);
+              },
+              child: const Text('Go to Second Screen'),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+```
+### Step 6. Create allfiles.dart. It contains all the exports of all other files. File used as one import to reduce the repetition of imports.
+```dart
+export 'package:flutter/material.dart';
+export 'firstscreen.dart';
+export 'Secondscreen.dart';
+export 'Thirdscreen.dart';
+```
+1. To run the flutter app use the following commands:
+```cmd
+flutter run -d chrome //runs the flutter app in chrome browser
+```
+or you can run using a browser
+```cmd
+flutter run -d edge //runs the flutter app in the edge browser
+```
+Below are screenshots showing the expected results
+
+![FirstPage](/engineering-education/introduction-of-routing-and-navigation-in-flutter-applications/firstpage.jpg)
+
+![SecondPage](/engineering-education/introduction-of-routing-and-navigation-in-flutter-applications/secondpage.jpg)
+
+
+
+![ThirdPage](/engineering-education/introduction-of-routing-and-navigation-in-flutter-applications/thirdpage.jpg)
+
+
+For any query Reachout @ [this GitHub repository](https://github.com/karehnikita/).
+
+### Conclusion
+
```

Comment:
The conclusion summarizes what the article has covered and the very key points. So you can fix that too.